fabulaws/library/wsgiautoscale/templates/nginx.conf

```diff
@@ -32,7 +32,9 @@ server {
         location = /healthcheck.html {
             # Turn off basic auth for healthcheck
             auth_basic "off";
-            proxy_pass http://django_server;
+            # Abbreviated version of logic below to serve healthcheck.html (override) if it exists
+            if (-f $request_filename) { break; }
+            if (!-f $request_filename) { proxy_pass http://django_server; break; }
```

Comment:
I don't remember why I did it this way originally; I actually tested your suggestion and it appears to work.

That said, I **might** have done it this way for a reason originally (the definitive copy of this logic is further down in the file) and since it's working at the moment I would prefer not to change it.

For further reference:
* https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/
* http://agentzh.blogspot.com/2011/03/how-nginx-location-if-works.html